CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- kube-rs: 0.74.0 -> 0.75.0 ([#490]).
+- `Client` methods now take the namespace as a `&str` (for namespaced resources) or `&()` (for cluster-scoped resources),
+  rather than always taking an `Option<&str>` ([#490]).
```

Comment:
This change could be marked as breaking so that the users of operator-rs are aware of the increased effort for upgrading.

```suggestion
- BREAKING: `Client` methods now take the namespace as a `&str` (for namespaced resources) or
  `&()` (for cluster-scoped resources), rather than always taking an `Option<&str>` ([#490]).
```